templates/org/team/teams.tmpl

```diff
@@ -37,7 +37,7 @@
 						{{end}}
 					</div>
 					<div class="ui bottom attached header">
-						<p class="team-meta">{{.NumMembers}} {{$.i18n.Tr "org.lower_members"}} · {{.NumRepos}} {{$.i18n.Tr "org.lower_repositories"}}</p>
+						<p class="team-meta">{{.NumMembers}} {{$.i18n.Tr "org.lower_members"}}&nbsp;·&nbsp;{{.NumRepos}} {{$.i18n.Tr "org.lower_repositories"}}</p>
```

Comment:
Width of `&nbsp;` will not be consistent across different fonts and I generally despise using whitespace for layout purposes, better to use something like

```html
<span class="mx-2">·</span>
```

Reply:
This goes without saying for me, but I was more focused on other stuff. I might revert this change in exchange to better solutions. `white-space: pre` exists at other places as well - I really can't see the use of that apart from supporting this type of weird layout technique with **breakable spaces** - ugh!